source/audio/appsVolume.py

```diff
@@ -163,7 +163,15 @@ def _toggleAppsVolumeState():
 	state = _APPS_VOLUME_STATES_ORDER[index]
 	config.conf["audio"]["applicationsVolumeMode"] = state.name
 	_updateAppsVolumeImpl(volume / 100.0, muted, state)
-	ui.message(state.displayString)
+	if state == AppsVolumeAdjusterFlag.ENABLED:
+		# Translators: Reported as a result of the command to allow or disallow to control the volume of other
+		# applications
+		msg = _("Allowed to control the volume of other applications")
```

Comment:
```suggestion
		msg = _("Controlling other applications' volume enabled")
```

Or, if you need the action word first:

```suggestion
		msg = _("Enabled control of other applications' volume")
```